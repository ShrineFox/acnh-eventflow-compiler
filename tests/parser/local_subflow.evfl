flow Root():
    TestActor1.Action0()
    run Local()

local flow Local():
    TestActor1.Action0()

