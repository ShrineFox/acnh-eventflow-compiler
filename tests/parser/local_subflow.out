RootNode[name=Root, vardefs=[], out_edges=[Event0]]
ActionNode[name=Event0, action=EventFlowActionAction0(), params={}, out_edges=[Event4]]
ActionNode[name=Event4, action=EventFlowActionAction0(), params={}, out_edges=[]]

